api/data_refinery_api/serializers.py

```diff
@@ -569,23 +569,22 @@ class DatasetDetailsExperimentSerializer(serializers.ModelSerializer):
     """ This serializer contains all of the information about an experiment needed for the download
     page
     """
-    organisms = serializers.StringRelatedField(many=True)
-    sample_metadata = serializers.ReadOnlyField(source='get_sample_metadata_fields')
-    pretty_platforms = serializers.ReadOnlyField()
+    organisms = serializers.ReadOnlyField(source='organism_names')
+    sample_metadata = serializers.ReadOnlyField(source='sample_metadata_fields')
```

Comment:
This change doesn't seem related to what the PR says it's about. What's the reason for this change?